ang/civicase/ActivityCard.js

```diff
@@ -1,56 +1,103 @@
-(function(angular, $, _) {
+(function (angular, $, _) {
+  var module = angular.module('civicase');
 
-  function activityCard($scope, getActivityFeedUrl, dialogService, templateExists) {
+  module.directive('caseActivityCard', function () {
+    return {
+      restrict: 'A',
+      templateUrl: '~/civicase/ActivityCard.html',
+      controller: caseActivityCardController,
+      scope: {
+        activity: '=caseActivityCard',
+        refresh: '=refreshCallback',
+        editActivityUrl: '=?editActivityUrl'
+      }
+    };
+  });
+
+  function caseActivityCardController ($scope, getActivityFeedUrl, dialogService, templateExists, crmApi) {
     var ts = $scope.ts = CRM.ts('civicase');
-    $scope.CRM = CRM;
     $scope.activityFeedUrl = getActivityFeedUrl;
+    $scope.CRM = CRM;
```

Comment:
We shouldn't populate the scope object. If we are not using `$scope.CRM` then this can be removed.